install.sh

```diff
@@ -32,20 +32,28 @@ if [ "$CM" = true ]
 then
     if [ "$VIRTUAL_ENV" != "" ]
     then
-        echo "Installing cm_client in virtualenv $VIRTUAL_ENV"
+        echo "The script will install cm_client in virtualenv $VIRTUAL_ENV"
     elif [ "$VIRTUAL_ENV" == "" ]
     then
-        echo "Installing cm_client in global scope"
+        echo "The script will install  cm_client in global scope"
     fi
-    pip install cm_client
-    status=$?
-    if [ $status -eq 0 ]
-    then
-        echo "Installation successful"
-    else
-        echo "Installation of cm_client module required for running the cluster registration script failed. Please install this manually and re-run the script"
-        exit $status        
-    fi 
+    while true; do
+        read -p "Do you wish to install cm_client  (y/n)?" choice
+        case "$choice" in
+            y|Y ) 
+            pip install cm_client
+            status=$?
+            if [ $status -eq 0 ]
+            then
+                echo "Installation successful"
+            else
+                echo "Installation of cm_client module required for running the cluster registration script failed. Please install this manually and re-run the script"
+                exit $status        
+            fi; break;;
+            n|N ) exit;;
```

Comment:
Perhaps some information here? Something along the lines of "cm_client is required for registering Cloudera Manager managed clusters. If you do not wish to work with those clusters, re-run the installation script after unsetting the CM variable"